<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>356</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>545</width>
    <height>356</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButtonCarps">
         <property name="text">
          <string>Carpas</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDorms">
         <property name="text">
          <string>Dormis</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditSearch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maxLength">
          <number>32767</number>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Buscar...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>545</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo_camping"/>
    <addaction name="actionAbrir_archivo_de_camping"/>
    <addaction name="actionGuardar_camping_como"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuCamping">
    <property name="title">
     <string>Camping</string>
    </property>
    <addaction name="actionCarpas_y_dormis"/>
    <addaction name="actionEditar_plantilla_para_el_recibo_del_cliente"/>
    <addaction name="separator"/>
    <addaction name="actionVer_carpas"/>
    <addaction name="actionVer_dormis"/>
   </widget>
   <widget class="QMenu" name="menuReportes">
    <property name="title">
     <string>Reportes</string>
    </property>
    <addaction name="actionLista_de_hu_spedes"/>
    <addaction name="actionLista_de_vencidos"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAyuda"/>
    <addaction name="actionPedir_soporte"/>
    <addaction name="separator"/>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuCamping"/>
   <addaction name="menuReportes"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
  <action name="actionNuevo_camping">
   <property name="text">
    <string>Nuevo camping</string>
   </property>
  </action>
  <action name="actionAbrir_archivo_de_camping">
   <property name="text">
    <string>Abrir archivo de camping</string>
   </property>
  </action>
  <action name="actionGuardar_camping_como">
   <property name="text">
    <string>Guardar camping como...</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionAyuda">
   <property name="text">
    <string>Manual de usuario</string>
   </property>
  </action>
  <action name="actionPedir_soporte">
   <property name="text">
    <string>Pedir soporte</string>
   </property>
  </action>
  <action name="actionCarpas_y_dormis">
   <property name="text">
    <string>Configurar carpas y dormis...</string>
   </property>
  </action>
  <action name="actionVer_carpas">
   <property name="text">
    <string>Ver carpas</string>
   </property>
  </action>
  <action name="actionVer_dormis">
   <property name="text">
    <string>Ver dormis</string>
   </property>
  </action>
  <action name="actionEditar_plantilla_para_el_recibo_del_cliente">
   <property name="text">
    <string>Editar plantilla para el recibo del cliente</string>
   </property>
  </action>
  <action name="actionLista_de_hu_spedes">
   <property name="text">
    <string>Lista de hu√©spedes</string>
   </property>
  </action>
  <action name="actionLista_de_vencidos">
   <property name="text">
    <string>Lista de vencidos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
